pkg/ui/v1alpha3/const.go

```diff
@@ -0,0 +1,30 @@
+package v1alpha3
+
+import (
+	"github.com/kubeflow/katib/pkg/util/v1alpha3/env"
+	"strings"
+)
+
+const (
+	// AvailableNameSpaceEnvName is the env name of available namespaces
+	AvailableNameSpaceEnvName = "KATIB_UI_AVAILABLE_NS"
+	// ClusterRoleKey is a key represents having access to cluster role
+	ClusterRoleKey = "KATIB_UI_CLUSTER_ROLE"
+)
+
+var (
+	availableNameSpaces []string
+	hasClusterRole      bool
+)
+
+func init() {
+	ns := env.GetEnvOrDefault(AvailableNameSpaceEnvName, ClusterRoleKey)
```

Comment:
Couple of questions here

1.  Is it right to get the namespaces during init function? What happens if env variable is changed later?
2.  GetEnvOrDefault has fallback value as the second argument.  Did you mean to keep ClusterRoleKey as the default value?

Reply:
@johnugeorge 
1.  To me, this env variable should be set via manifest yaml file. I can't see any motivation to modify this env variable later.
2. Yes.